fvm/evm/stdlib/contract.cdc

```diff
@@ -206,4 +214,62 @@ contract EVM {
 
         return InternalEVM.decodeABI(types: types, data: data)
     }
+
+    /// validateCOAOwnershipProof validates a COA ownership proof
+    access(all)
+    fun validateCOAOwnershipProof(
+        address: Address,
+        path: PublicPath,
+        signedData: [UInt8],
+        keyIndices: [UInt64],
+        signatures: [[UInt8]],
+        evmAddress: [UInt8; 20]
```

Comment:
random thought, but I'm seeing this array definition multiple times and I think we should maybe create an `evmAddress` type for it